core/templates/base-components/oppia-footer.component.html

```diff
@@ -1,7 +1,37 @@
 <footer class="oppia-footer" role="contentinfo">
+  <div class="oppia-footer-newsletter-container">
+    <div class="oppia-footer-newsletter">
+      <div class="oppia-newsletter-content">
+        <h2 [innerHTML]="'I18N_FOOTER_NEWSLETTER_HEADING' | translate"></h2>
```

Comment:
```suggestion
        <h2>{{ 'I18N_FOOTER_NEWSLETTER_HEADING' | translate }}</h2>
```
Use data interpolation instead of `innerHTML` when the variable is a text. Here and elsewhere.

Reply:
Done